src/plugins/Transloadit/index.js

```diff
@@ -486,6 +707,9 @@ module.exports = class Transloadit extends Plugin {
       this.uppy.on('upload-success', this.onFileUploadURLAvailable)
     }
 
+    this.uppy.on('restore:get-data', this.getPersistentData)
+    this.uppy.on('restored', this.onRestored)
+
```

Comment:
So will `restore:` here a goldenretriever-specific event and `restored` a global one?

Reply:
sorta. my thinking is that while `restore:get-data` is used to communicate between different plugins, Core doesn't use it. other plugins use it to give more data to GoldenRetriever when it stores restore data.